core/arch/arm/kernel/kern.ld.S

```diff
@@ -58,18 +58,31 @@
 #define ROUNDDOWN(x, y)		(((x) / (y)) * (y))
 #endif
 
+/*
+ * TEE_RAM_VA_START:            The start virtual address of the TEE RAM
+ * TEE_TEXT_VA_START:           The start virtual address of the OP-TEE text
+ */
+#define TEE_RAM_VA_START                CFG_TEE_RAM_START
+#define TEE_TEXT_VA_START               (TEE_RAM_VA_START + \
+			(CFG_TEE_LOAD_ADDR - CFG_TEE_RAM_START))
+
 OUTPUT_FORMAT(CFG_KERN_LINKER_FORMAT)
 OUTPUT_ARCH(CFG_KERN_LINKER_ARCH)
 
 ENTRY(_start)
 SECTIONS
 {
-	. = CFG_TEE_LOAD_ADDR;
-
+	. = TEE_TEXT_VA_START;
+#ifdef ARM32
+	ASSERT(!(TEE_TEXT_VA_START & 31), "text start should align to 0x20")
```

Comment:
minor:
s/should/must/
s/0x20/32 bytes/

Reply:
OK